makepanda/makepanda.py

```diff
@@ -5208,6 +5209,16 @@ def CreatePandaVersionFiles():
   TargetAdd('libp3daeegg.lib', opts=['FCOLLADA', 'CARBON'])
 
 #
+# DIRECTORY: pandatool/src/assimp
+#
+if (PkgSkip("PANDATOOL") == 0 and PkgSkip("ASSIMP")==0):
+  OPTS=['DIR:pandatool/src/assimp', 'ASSIMP']
```

Comment:
Since this is a plug-in, please add 'MODULE' to OPTS in order to prevent it from appending a soname suffix such as libp3assimp.so.1.9.0